bibtex-actions.el

```diff
@@ -259,50 +321,58 @@ key associated with each one."
            bibtex-actions-template-suffix))
          (main-width (truncate (* (frame-width) 0.65)))
          (suffix-width (truncate (* (frame-width) 0.34))))
-    (cl-loop
-     for candidate in (bibtex-completion-candidates)
-     collect
-     (let* ((pdf (if (assoc "=has-pdf=" (cdr candidate)) " has:pdf"))
-            (note (if (assoc "=has-note=" (cdr candidate)) "has:note"))
-            (link (if (or (assoc "doi" (cdr candidate))
-                          (assoc "url" (cdr candidate))) "has:link"))
-            (citekey (bibtex-completion-get-value "=key=" candidate))
-            (candidate-main
-             (bibtex-actions--format-entry
-              candidate
-              main-width
-              main-template))
-            (candidate-suffix
-             (bibtex-actions--format-entry
-              candidate
-              suffix-width
-              suffix-template))
-            ;; We display this content already using symbols; here we add back
-            ;; text to allow it to be searched, and citekey to ensure uniqueness
-            ;; of the candidate.
-            (candidate-hidden (s-join " " (list pdf note link context citekey))))
-       (cons
-        ;; If we don't trim the trailing whitespace, 'completing-read-multiple' will
-        ;; get confused when there are multiple selected candidates.
-        (s-trim-right
-         (concat
-          ;; We need all of these searchable:
-          ;;   1. the 'candidate-main' variable to be displayed
-          ;;   2. the 'candidate-suffix' variable to be displayed with a different face
-          ;;   3. the 'candidate-hidden' variable to be hidden
-          (propertize candidate-main 'face 'bibtex-actions-highlight) " "
-          (propertize candidate-suffix 'face 'bibtex-actions) " "
-          (propertize candidate-hidden 'invisible t)))
-        citekey)))))
+    (cl-loop for candidate being the hash-values of (parsebib-parse files)
+             collect
+             (let* ((files
+                     (when (bibtex-actions--files-for-key
+                            (bibtex-actions-get-value "=key=" candidate)
+                            bibtex-actions-library-paths bibtex-actions-file-extensions)
+                           " has:files"))
```

Comment:
```suggestion
                     (when (or (assoc "=has-pdf=" (cdr candidate))
                          (bibtex-actions--files-for-key
                            (bibtex-actions-get-value "=key=" candidate)
                            bibtex-actions-library-paths bibtex-actions-file-extensions)
                           " has:files"))
```

Reply:
Probably, but then should use `bibtex-actions-get-value` there, since that will properly deal with the tex formats and json.

Reply:
Actually, this isn't right. Need to check on this.